config.yaml

```diff
@@ -4,8 +4,14 @@ title: gRPC
 disableKinds: [taxonomy, taxonomyTerm]
 pygmentsCodeFences: true
 
+theme: ['docsy']
+
+enableGitInfo: true
+
 params:
-  repo: https://github.com/grpc/grpc.io
```

Comment:
The `repo` param is still being used (deleting it breaks the links in pages like `404`), so please revert this delete.

I'd also suggest you define an anchor that can be used to define `github_repo` and `github_project_repo` too, something like this:

```yaml
  repo: &repo https://github.com/grpc/grpc.io
  ...
  github_repo: *repo
  github_project_repo: *repo
```

Reply:
Fixed!